app.yaml

```diff
@@ -47,12 +47,21 @@ handlers:
   # NB: not minified. TODO(sll): fix.
   static_dir: core/templates/dev/head/css
   secure: always
+- url: /fonts
```

Comment:
I don't think you need to expose a separate handler for this, actually -- let's just use /third_party/generated/fonts and not do a redirect.